styles/globals.scss

```diff
@@ -254,6 +253,9 @@
   margin-right: auto;
   margin-left: auto;
 }
+.mb-17 {
```

Comment:
write this styling in Blog component styling file, and minus margin is not good way for fixing